packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Snippets/Snippet.cs

```diff
@@ -99,12 +99,12 @@ public static InvokeMethodExpression Invoke(this ParameterProvider parameter, st
         public static ValueExpression Property(this ParameterProvider parameter, string propertyName, bool nullConditional = false)
             => new MemberExpression(nullConditional ? new NullConditionalExpression(parameter) : parameter, propertyName);
 
-        public static ValueExpression Invoke(this FieldProvider field,
+        public static ValueExpression Invoke(this PropertyProvider property,
```

Comment:
I think its fine to have both extensions, we will most likely use both eventually